runner/call_template_data.go

```diff
@@ -18,7 +18,7 @@ var seededRand *rand.Rand = rand.New(
 	rand.NewSource(time.Now().UnixNano()))
 
 // call template data
-type callTemplateData struct {
+type callData struct {
```

Comment:
I think this needs to be exposed as public struct, `CallData`.

Reply:
You are right, we planned to make them public. 👍